release_notes/ocp-4-14-release-notes.adoc

```diff
@@ -101,6 +101,11 @@ User-defined tags for Microsoft Azure was previously introduced as Technology Pr
 
 You can now configure user-defined labels and tags in Google Cloud Platform (GCP) for grouping resources and for managing resource access and cost. User-defined labels can be applied only to resources created by {product-title} installation program and its core components. User-defined tags can be applied only to resources created by the {product-title} Image Registry Operator. For more information, see xref:../installing/installing_gcp/installing-gcp-customizations.adoc#installing-gcp-user-defined-labels-and-tags_installing-gcp-customizations[Managing the user-defined labels and tags for GCP].
 
+[id="ocp-4-14-installing-ocp-azure-restricted-networks"]
+==== Installing an {product-title} cluster on Microsoft Azure in a restricted network
+
+In {product-title} 4.14, you can install a cluster on Microsoft Azure in a restricted network for installer-provisioned infrastructure and user-provisioned infrastructure. For installer-provisioned infrastructure, you can create an internal mirror of the installation release content on an existing Azure Virtual Network (VNet). For user-provisioned infrastructure, you can install a cluster on Microsoft Azure by using infrastructure that you provide. For more information, see xref:../installing/installing_azure/installing-restricted-networks-azure-installer-provisioned.adoc#installing-restricted-networks-azure-installer-provisioned[Installing a cluster on Azure in a restricted network] and xref:../installing/installing_azure/installing-restricted-networks-azure-user-provisioned.adoc#installing-restricted-networks-azure-user-provisioned[Installing a cluster on Azure in a restricted network with user-provisioned infrastructure].
```

Comment:
Just a suggestion. You could use IPI and UPI abbreviations and use them in the following sentences.